src/app-layout/interfaces.ts

```diff
@@ -114,12 +114,14 @@ export interface AppLayoutProps extends BaseComponentProps {
 
   /**
    * Top area of the page content.
+   * **Deprecated**, replaced by the content layout `header` slot.
```

Comment:
"content layout" might not have a clear meaning for users who see this. I would suggest to explicitly mention that there is a component with that name. For example:
> **Deprecated**, replaced by the `header` slot of the ContentLayout component.

(same applies to the `disableContentHeaderOverlap` property)